### PR TITLE
[pt] Add /content/pt/docs/concepts/context-propagation.md

### PR DESCRIPTION
**Pull request by EzzioMoreira**
Add a localization to the context-propagator.md.

Issue: https://github.com/open-telemetry/opentelemetry.io/issues/4922

### CODE REVIEW

**Comment by edsoncelio** [comment]
Preview: https://deploy-preview-4984--opentelemetry.netlify.app/

File: content/pt/docs/concepts/context-propagation.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: Propagação de contexto
+weight: 10
+description: Entenda os conceito que torna possível o Rastreamento Distribuído.
```
Comment (edsoncelio) [inline content/pt/docs/concepts/context-propagation.md:4]:
```suggestion
description: Entenda os conceitos que tornam possível o Rastreamento Distribuído.
```

File: content/pt/docs/concepts/context-propagation.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+default_lang_commit: 2bda479b6ce77f51266845ade9fe1b431dfde0d3
+---
+
+Com a propagação de contexto, os [Sinais](/docs/concepts/signals) podem ser
+correlacionados entre si, independentemente de onde são gerados. Embora não se
+limite ao rastreamento, a propagação de contexto permite que os
+[rastros](/docs/concepts/signals/traces) criem informações causais sobre um
+sistema que são distribuídos arbitrariamente entre processos e limites de rede.
```
Comment (edsoncelio) [inline content/pt/docs/concepts/context-propagation.md:12]:
```suggestion
sistema que é distribuído arbitrariamente entre processos e limites de rede.
```

**Review by edsoncelio — COMMENTED** [review]
Só alguns detalhes, mais um PR muito bom @EzzioMoreira!

**Comment by edsoncelio** [comment]
LGTM!

**Review by svrnm — APPROVED** [review]

**Pull request merged**